### DISCUSSION TITLE
Matrix or Telegram

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#283] [Polls]
We currently have a Telegram group (https://t.me/Kvaesitso). There has been a request to create a Matrix room as well. Which one do you prefer?

**Comment by shtrophic** [comment]
27 Votes, probably not representative at all. But anyway, are you still considering this?

**Comment by Dayanandhan24** [comment]
What is Matrix?

**Reply by shtrophic** [reply]
Short answer: a free and open chat protocol for decentralized communication; In principle similar to Mastodon or Lemmy which implement ActivityPub. In practice, every user gets to choose their favorite app and server, while still being able to chat with other users using different apps from different servers. Multiple other github projects do their comms this way. You can refer to: https://matrix.org/try-matrix/

Long answer: https://matrix.org/about/
